tests/challenges/switch6b.gno

```diff
@@ -0,0 +1,18 @@
+package main
```

Comment:
could you move those challenges to the appropriate folder (outside of `challenges`)?

they'll become official tests and be included in the CI.

Reply:
Should I duplicate those files between `files` and `files2` folder? Or only `files` folder ? 

TBH I don't really get why there's this separation.

Reply:
The difference is:
* `files` uses `ImportModeNativePreferred`
* `files2` uses `ImportModeStdlibsPreferred`

See: https://github.com/gnolang/gno/blob/master/tests/imports.go#L61-L63

---

In your case, without any import, `files/` should be enough, but it's cheap to also add it in `files2/` and can prevent eventual future bugs. So, let's do both.

Reply:
TestChallenges is a good addition, but it should be disabled by default; let's make it toggleable with an env var or something. It's just for advanced developers working on this need.

Goals are:
* CI is green
* running `make test` or `go test ./...` should also work by default

Let's keep the exception just for advanced users.